packages/deployments/contracts/deploy/01_deployMessaging.ts

```diff
@@ -111,11 +111,9 @@ const handleDeployHub = async (
       continue;
     }
 
-    const contract = getConnectorName(protocol, mirrorChainId);
-    if (
-      (!connectorName.includes("Optimism") && !connectorName.includes("Polygon")) ||
-      connectorName.includes("Mainnet")
-    ) {
+    const contract = getConnectorName(protocol, mirrorChainId, protocol.hub);
+    console.log(contract);
+    if ((!contract.includes("Optimism") && !contract.includes("Polygon")) || contract.includes("Mainnet")) {
```

Comment:
a really helpful update for this would be to put these names in some kind of constant... we'll want to eventually make it so these names are less hardcoded in the future...